lib/standalone.mjs

```diff
@@ -83,6 +84,15 @@ export default async function standalone({
 	let formatterFunction;
 
 	try {
+		/** @see https://github.com/stylelint/stylelint/issues/7447 */
+		if (formatter === 'github') {
+			emitDeprecationWarning(
+				'"github" formatter is deprecated',
+				'GITHUB_FORMATTER',
+				'Please use "stylelint-actions-formatters" instead.',
+			);
+		}
+
```

Comment:
minor
refactor

I think this should be moved in `getFormatterFunction` inside the first `if`.